client/src/app/site/agenda/services/agenda-repository.service.ts

```diff
@@ -0,0 +1,79 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+
+import { BaseRepository } from '../../base/base-repository';
+import { DataStoreService } from '../../../core/services/data-store.service';
+import { Item } from '../../../shared/models/agenda/item';
+import { ViewItem } from '../models/view-item';
+import { AgendaBaseModel } from '../../../shared/models/base/agenda-base-model';
+import { BaseModel } from '../../../shared/models/base/base-model';
+
+/**
+ * Repository service for users
+ *
+ * Documentation partially provided in {@link BaseRepository}
+ */
+@Injectable({
+    providedIn: 'root'
+})
+export class AgendaRepositoryService extends BaseRepository<ViewItem, Item> {
+    public constructor(DS: DataStoreService) {
+        super(DS, Item);
+    }
+
+    /**
+     * Returns the corresponding content object to a given {@link Item} as an {@link AgendaBaseModel}
+     * @param agendaItem
+     */
+    public getContentObject(agendaItem: Item): AgendaBaseModel {
```

Comment:
Please make private if it isn't used anywhere else.

Reply:
marked private. It's only used there